tests/Behat/ProjectContext.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Behat;
+
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+use Drupal\user\Entity\Role;
+
+/**
+ * Provide steps to test the project content type display.
+ */
+class ProjectContext extends RawDrupalContext {
+
+  /**
+   * Grant all necessary permissions so that anonymous users can see projects.
+   *
+   * @Given anonymous users can see projects
+   */
+  public function anonymousUsersCanSeeEvents(): void {
+    user_role_grant_permissions(Role::ANONYMOUS_ID, [
```

Comment:
@ademarco, I would prefer to add to the vendor/drupal/drupal-extension/src/Drupal/DrupalExtension/Context/DrupalContext.php statement "I am an anonymous user with the :permissions permission(s)". It could be useful for oe_theme/tests/Behat/EventContext.php also (and for future needs)